pages/project-details/[id]/index.js

```diff
@@ -125,6 +141,9 @@ export default function ProjectDetails({
         }
       />
       <Main>
+        <Button onClick={toggleActiveStatus} isActive={currentProject.isActive}>
```

Comment:
tiny thing, but i would specifiy the button type=button, since the default button type is submit

Reply:
coding 101... thank you 🙂